src/phasorpy/phasor.py

```diff
@@ -391,78 +392,231 @@ def phasor_semicircle(
 def phasor_calibrate(
     real: ArrayLike,
     imag: ArrayLike,
-    phase0: ArrayLike = 0.0,
-    modulation0: ArrayLike = 1.0,
+    reference_real: ArrayLike,
+    reference_imag: ArrayLike,
     /,
+    frequency: ArrayLike,
+    lifetime: ArrayLike,
+    *,
+    fraction: ArrayLike | None = None,
+    preexponential: bool = False,
+    unit_conversion: float = 1e-3,
+    method: Literal['mean', 'median'] = 'mean',
 ) -> tuple[NDArray[Any], NDArray[Any]]:
-    """Return calibrated/referenced phasor coordinates.
+    """
+    Return calibrated/referenced phasor coordinates.
 
     Calibration of phasor coordinates in fluorescence lifetime analysis is
     necessary to account for the instrument response function (IRF) and delays
     in the electronics.
 
-    This function can also be used to transform/rotate any phasor coordinate.
+    Parameters
+    ----------
+    real : array_like
+        Real component of phasor coordinates to be calibrated.
+    imag : array_like
+        Imaginary component of phasor coordinates to be calibrated.
+    reference_real : array_like
+        Real component of phasor coordinates from reference of known lifetime.
+        Must be measured with the same instrument setting as the phasor
+        coordinates to be calibrated.
+    reference_imag : array_like
+        Imaginary component of phasor coordinates from reference of known
+        lifetime.
+        Must be measured with the same instrument setting as the phasor
+        coordinates to be calibrated.
+    frequency : array_like
+        Laser pulse or modulation frequency in MHz.
+        A scalar or one-dimensional sequence.
+    lifetime : array_like
+        Lifetime components in ns. See notes below for allowed dimensions.
```

Comment:
`See notes below for allowed dimensions.` -> `Must be scalar or one dimensional.`